src/NServiceBus.Gateway.AcceptanceTests/When_sending_fails_with_retries.cs

```diff
@@ -163,5 +170,38 @@ public Task Handle(AnyMessage errorMessage, IMessageHandlerContext context)
                 Context testContext;
             }
         }
+
+        class FaultyChannelSender : IChannelSender
```

Comment:
Would it make sense to extract those impls to a root folder and reuse it in as well ?

https://github.com/Particular/NServiceBus.Gateway/pull/54/files#diff-8910b6948833c6ad5dcd6fd9e87e7e9eR120